src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -511,6 +511,21 @@
           Condition="'@(_InnerPackageCacheFiles)' != ''" />
   </Target>
 
+  <Target Name="CopyDotNetBuildArtifactsLogs"
+        Condition="'$(IsUtilityProject)' != 'true' and
+                    Exists('$(ProjectDirectory)artifacts')">
+
```

Comment:
styling nit: remove the empty line after the Target opening tag

```suggestion
```

Reply:
Got it, removed the empty line.